src/Tags/Mount.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Statamic\Tags;
+
+use Statamic\Facades\Collection;
+
+class Mount extends Tags
+{
+    /**
+     * {{ mount:* }}.
+     */
+    public function wildcard($tag)
+    {
+        return $this->mount($tag);
+    }
+
+    /**
+     * {{ mount handle="" }}.
+     */
+    public function index()
+    {
+        return $this->mount($this->params->get('handle'));
+    }
+
+    private function mount($handle)
+    {
+        if (! $collection = Collection::find($handle)) {
+            return;
+        }
+
+        return $collection->url();
```

Comment:
use Statamic\Facades\Site;

`$collection->url(Site::current()->handle())`

To get the localized version.

Reply:
Cheers Rob, I'll update it.

Reply:
I've updated it and the test.

Reply:
Ah thanks!

Reply:
Lovely!